Library/Homebrew/utils.rb

```diff
@@ -291,6 +291,18 @@ def paths
   end.uniq.compact
 end
 
+# return the shell profile file based on users' preference shell
+def shell_profile
+  case ENV["SHELL"]
+  when %r{/bash} then "~/.bash_profile"
+  when %r{/zsh} then "~/.zshrc"
+  when %r{/t?csh} then "~/.cshrc"
+  when %r{/ksh} then "~/.kshrc"
```

Comment:
You want something like this for the ` csh ` shells:

` echo 'setenv PATH #{HOMEBREW_PREFIX}/sbin:$PATH' >> ~/.cshrc `